AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/actions.js

```diff
@@ -378,7 +378,13 @@ define([
                      * On error
                      */
                     error: function (response) {
-                        messages.add('error', response.responseJSON.message);
+                        var errorMessage = response.JSON ? response.JSON.meassage : '';
```

Comment:
I think it's better to use a default error message instead of empty string
```suggestion
                        var errorMessage = response.JSON ? response.JSON.meassage : 'Failed to fetch licensing information.';
```